    else:
        print(f"attempted {num+1} times and failed")
